### PR TITLE
replaced localStorage with settingsStore method

### LINKED ISSUE #1171: Some file handling keys in local storage do not have the kiwixjs- prefix
@Rishabhg71 All keys in localStorage must have the kiwixjs- prefix to avoid conflict with other applications that might use the same keys (e.g. the Kiwix PWA), but I've just noticed two in there, relating to File System Access, that are bare keys (see screenshot). This means your code must be writing directly to, and reading directly from, localStorage instead of using the settingsStore method. It's important always to use settingsStore unless there is a strong reason to write directly to localStorage. SettingsStore manages the prefix.

Would you be able to fix the code that writes these values, please? I'm sorry I didn't notice this at the time of the PR.

image

### PR DESCRIPTION
**Pull request by Greeshmanth1909**
resolves #1171
As the title says, replaced localStorage method with settingsStore method.
Heres a list of tests I performed:
- ran `npm run serve` and `npm run preview` and tested for chrome and firefox in both Jquery and serviceWorker modes.
- couldn't try the app in IE mode as I do not have a windows machine
- ran `npm test` all tests were successful
- tested the extension in developer mode on chrome
- tested the firefox extension

In all the tests, I was able to open the application and view .zim files. I also checked the localStorage and verified the presence of `kiwixjs-` prefix.

### CODE REVIEW

**Comment by Jaifroid** [comment]
@Greeshmanth1909 Thanks for your PR. I'll review shortly.

**Comment by Greeshmanth1909** [comment]
what went wrong and how can I replicate this error?

**Comment by Jaifroid** [comment]
Probably just a temporary error. Tests can be a bit unstable if there's a lot of load on the system. I'll re-run.

**Comment by Greeshmanth1909** [comment]
Got it, thanks!

**Comment by Greeshmanth1909** [comment]
@Jaifroid Are there any more tests?

**Comment by Jaifroid** [comment]
> @Jaifroid Are there any more tests?

No, just a few manual ones from me. I'll do those now.

**Comment by Jaifroid** [comment]
Just to confirm that it works fine in IE11.

**Comment by Jaifroid** [comment]
My other manual tests are all fine. I'll review code now.

File: www/js/lib/abstractFilesystemAccess.js [hunk 1]
```diff
@@ -234,7 +235,7 @@ function loadPreviousZimFile () {
     // It's a bit hacky but it works and I am not sure if there is any other way ATM
     setTimeout(() => {
         if (window.params.isFileSystemApiSupported || window.params.isWebkitDirApiSupported) {
-            const filenames = localStorage.getItem('zimFilenames');
+            const filenames = settingsStore.getItem('zimFilenames');
             if (filenames) updateZimDropdownOptions(filenames.split('|'), '');
         }
     }, 200);
```
Comment (Jaifroid) [inline www/js/lib/abstractFilesystemAccess.js:241]: This is just a note to self, not part of this PR, but this timeout may well be the problem with inconsistency in loading the contents of archives. It introduces a race condition, and needs looking at as a separate issue.

**Review by Jaifroid — APPROVED** [review]
The changes all look good, many thanks @Greeshmanth1909! Also many thanks for listing all the testing you've done, it really helps to know where I need to concentrate my own testing. Please ignore the one comment on the code, this is just a note to self about a possible separate issue unrelated to this PR.

Let me know when you're happy for me to squash/merge. You will appear as the PR author.

**Comment by Greeshmanth1909** [comment]
@Jaifroid Thats great! Please go ahead and merge the PR. 
Thank you!
